src/main/kotlin/no/nav/syfo/aksessering/api/SykmeldingStatusApi.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.syfo.aksessering.api
+
+import io.ktor.application.call
+import io.ktor.http.HttpStatusCode
+import io.ktor.request.receive
+import io.ktor.response.respond
+import io.ktor.routing.Route
+import io.ktor.routing.post
+import no.nav.syfo.aksessering.SykmeldingService
+import no.nav.syfo.persistering.StatusEvent
+import no.nav.syfo.persistering.SykmeldingStatusEvent
+
+fun Route.registerSykmeldingStatusApi(sykmeldingService: SykmeldingService) {
+
+    post("/sykmeldinger/{sykmeldingsid}/status") {
+        val sykmeldingId = call.parameters["sykmeldingsid"]!!
+        val sykmeldingStatusEventDTO = call.receive<SykmeldingStatusEventDTO>()
+        val sykmeldingStatusEvent = SykmeldingStatusEvent(
+                sykmeldingId,
+                sykmeldingStatusEventDTO.timestamp,
+                sykmeldingStatusEventDTO.statusEvent.toStatusEvent())
+        sykmeldingService.registrerStatus(sykmeldingStatusEvent)
+        call.respond(HttpStatusCode.OK)
+    }
+}
+
+private fun StatusEventDTO.toStatusEvent(): StatusEvent {
+    return when (this) {
```

Comment:
Når jeg ser dette skulle jeg ønske det var på norsk.. Når det er på engelsk må vi jo oversette det mentalt hver gang vi snakker med noen (og hverandre), og jeg er redd for at det vil føre til misforståelser. 

Vi bør forresten også ha en status som reflekterer at vi ikke ønsker at den skal vises for bruker, i syfoservice heter den SLETTET (brukes for det vi ønsker å slette, men siden vi aldri sletter noe setter vi heller den statusen).